### PR TITLE
Updated dependencies and did pom cleanup as required by Maven

### PR DESCRIPTION
**Pull request by mehmetg**
Pre-release clean up.

### CODE REVIEW

File: testng/pom.xml [hunk 1]
```diff
@@ -30,11 +30,6 @@
             <version>2.6</version>
         </dependency>
         <dependency>
-            <groupId>com.saucelabs</groupId>
-            <artifactId>sauce_java_common</artifactId>
-            <version>${project.version}</version>
-        </dependency>
-        <dependency>
```
Comment (halkeye) [inline testng/pom.xml]: was this intentional?

Comment (mehmetg) [inline testng/pom.xml]: Yes, it was included twice and maven warned about unstable builds indicating it'd fail in the next revision on maven.

**Comment by halkeye** [comment]
then LGTM

**Comment by mehmetg** [comment]
ty!

**Pull request merged**